client/src/layout/MainLayout.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import NavBar from '../components/NavBar.jsx';
+
+const MainLayout = ({children}) => {
```

Comment:
Maybe it's a React thing, but where are children coming from?

Reply:
It is a React Router thing! Each time I wrap Main Layout around other components, it passes whatever I've wrapped into MainLayout as children, which in this use case makes it so the display shows the NavBar on top and then the rest of the page below it. Here's a screenshot of the pending About page to show the way Main Layout gets wrapped around the content so that React Router knows what to display!

![Screenshot 2024-10-30 at 3 18 25 PM](https://github.com/user-attachments/assets/39913922-52cf-46c1-baa0-11d2ba6a143b)